docs/modules/ROOT/pages/api/test_utilities.adoc

```diff
@@ -0,0 +1,317 @@
+= Test Utilities
+
+This module provides various helper functions for declaring, deploying,
+and testing Smart Contracts using Starknet Foundry.
+
+```cairo
+use openzeppelin_test_utils;
+```
+
+[.contract-index]
+.Members
+--
+.Inner modules
+* xref:#testutils-common[`++common++`]
+* xref:#testutils-deployment[`++deployment++`]
+* xref:#testutils-events[`++events++`]
+* xref:#testutils-signing[`++signing++`]
+--
+
+[#testutils-Inner-Modules]
+==== Inner modules
+
+[.contract-item]
+==== `[.contract-item-name]#++common++#` [.item-kind]#module#
+
+See xref:#testutils-common[`openzeppelin_test_utils::common`].
+
+[.contract-item]
+==== `[.contract-item-name]#++deployment++#` [.item-kind]#module#
+
+See xref:#testutils-deployment[`openzeppelin_test_utils::deployment`].
+
+[.contract-item]
+==== `[.contract-item-name]#++events++#` [.item-kind]#module#
+
+See xref:#testutils-events[`openzeppelin_test_utils::events`].
+
+[.contract-item]
+==== `[.contract-item-name]#++signing++#` [.item-kind]#module#
+
+See xref:#testutils-signing[`openzeppelin_test_utils::signing`].
+
+[.contract]
+[[testutils-common]]
+=== `++common++`
+
+```cairo
+use openzeppelin_test_utils::common;
+```
+
+[.contract-index]
+.Members
+--
+.Functions
+* xref:#testutils-common-panic_data_to_byte_array[`++panic_data_to_byte_array(panic_data)++`]
+* xref:#testutils-common-to_base_16_string[`++to_base_16_string(value)++`]
+* xref:#testutils-common-assert_entrypoint_not_found_error[`++assert_entrypoint_not_found_error(result, selector, contract_address)++`]
+
+.Traits
+* xref:#testutils-common-IntoBase16StringTrait[`++IntoBase16StringTrait++`]
+--
+
+[#testutils-common-Functions]
+==== Functions
+
+[.contract-item]
+[[testutils-common-panic_data_to_byte_array]]
+==== `[.contract-item-name]#++panic_data_to_byte_array++#++(panic_data: Array<felt252>) → ByteArray++` [.item-kind]#function#
+
+Converts panic data into a string (ByteArray).
+
+`panic_data` is expected to be a valid serialized byte array with an extra felt252 at the beginning, which is the BYTE_ARRAY_MAGIC.
+
+[.contract-item]
+[[testutils-common-to_base_16_string]]
+==== `[.contract-item-name]#++to_base_16_string++#++(value: felt252) → ByteArray++` [.item-kind]#function#
+
+Converts a felt252 to a base 16 string padded to 66 characters (including the `0x` prefix).
```

Comment:
We're not highlighting `felt252` here but we are highlighting it below in `IntoBase16StringTrait`

Reply:
Added highlighting